vendor/github.com/dlclark/regexp2/README.md

```diff
@@ -92,6 +92,66 @@ if isMatch, _ := re.MatchString(`Something to match`); isMatch {
 
 This feature is a work in progress and I'm open to ideas for more things to put here (maybe more relaxed character escaping rules?).
 
+## Catastrophic Backtracking and Timeouts
+
+`regexp2` supports features that can lead to catastrophic backtracking.
+`Regexp.MatchTimeout` can be set to to limit the impact of such behavior; the
+match will fail with an error after approximately MatchTimeout. No timeout
+checks are done by default.
+
+Timeout checking is not free. The current timeout checking implementation starts
+a background worker that updates a clock value approximately once every 100
+milliseconds. The matching code compares this value against the precomputed
+deadline for the match. The performance impact is as follows.
```

Comment:
This is an indirect dependency used by goja, and `MatchTimeout` is not used there. I guess this is up to Dmitry to decide whether it makes sense, but it could have an impact for us if he decides to enable it.

Reply:
Yes - it is, but goja does not update it as regularly as I would prefer and we already have updated in the past to get some *unlikely* fixes.

This current update does not matter to use but removes the need to review this change in the future.